packages/app/pages/course/[cid]/insights.tsx

```diff
@@ -1,4 +1,5 @@
-import React, { ReactElement, useState } from "react";
+import React, { ReactElement } from "react";
```

Comment:
Instead of commenting this all out, can we just redirect to a 404 for a bit lmao?idk

Reply:
that sounds like a plan

Reply:
nevermind, we decided having the message saying insights is temporarily disabled would be a nice to have